GVFS/GVFS.Platform.Windows/WindowsPhysicalDiskInfo.cs

```diff
@@ -107,6 +107,11 @@ public class WindowsPhysicalDiskInfo
                     result.Add("VolumeSizeRemaining", FetchValue(mbo, "SizeRemaining"));
                 }
 
+                if (sizeStatsOnly)

```

Comment:
Not all your doing, but this method is quite long, does two separate things, and now with an exit in the middle, can be pretty hard to follow. I would split it into two new methods, so that this top level method can call the first, then early exit if needed, and then call the second.